
Turn 2
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 14, in <module>
    bot.run()
  File "D:\CNN project\SVM\SVMBot.py", line 83, in run
    command_queue.append(ship.move(game_map.get_safe_move(game_map[ship.position], game_map[di])))
  File "D:\CNN project\SVM\hlt\__init__.py", line 76, in move
    raw_direction = Direction.convert(direction)
  File "D:\CNN project\SVM\hlt\positionals.py", line 41, in convert
    raise IndexError
IndexError

